### PR TITLE
feat: Add tool to benchmark RLN proof generation/verification

### PR DESCRIPTION
**Pull request by alrevuelta**
Add simple tool to benchmark RLN proof generation and verification.

Run as:
```
make benchmarks LOG_LEVEL=NOTICE
./build/benchmarks
```

Output:
```
Proof generation times: 85 milliseconds, 723 microseconds, and 313 nanoseconds
Proof verification times: 2 milliseconds, 707 microseconds, and 137 nanoseconds
```

### CODE REVIEW

File: apps/benchmarks/benchmarks.nim [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import
+  math,
+  std/sequtils,
+  stew/results,
+  options,
+  ../../waku/waku_rln_relay/protocol_types,
```
Comment (rymnc) [inline apps/benchmarks/benchmarks.nim:6]: maybe we can just import waku_rln_relay?

File: apps/benchmarks/benchmarks.nim [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+manager.init()
+
+let data: seq[byte] = newSeq[byte](1024)
+
+var proofGenTimes: seq[times.Duration] = @[]
+var proofVerTimes: seq[times.Duration] = @[]
+
+for i in 0 .. 50:
```
Comment (rymnc) [inline apps/benchmarks/benchmarks.nim:34]: in a future iteration can we have this configurable?

**Review by rymnc — APPROVED** [review]
LGTM

File: apps/benchmarks/benchmarks.nim [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  var time = getTime()
```
Comment (Ivansete-status) [inline apps/benchmarks/benchmarks.nim:35]: In case we want to measue CPU-bounded tasks, it would be interesting to use `cpuTime` instead.

**Review by Ivansete-status — APPROVED** [review]
Awesome! Thanks!

File: apps/benchmarks/benchmarks.nim (hunk 2 above)
Comment (alrevuelta) [reply apps/benchmarks/benchmarks.nim:34]: imho no need to complicate things. this is intended for devs so just change the value and recompile :)

**Pull request merged**